core/os/thread.cpp

```diff
@@ -106,5 +109,6 @@ Thread::~Thread() {
 		thread.detach();
 	}
 }
+#endif
```

Comment:
```suggestion
#endif // THREADS_ENABLED
```

Reply:
Thanks! I completely forgot to add this one.